src/frontend/src/components/flowToolbarComponent/index.tsx

```diff
@@ -66,27 +67,39 @@ export default function FlowToolbar(): JSX.Element {
         open={openShareModal}
         setOpen={setOpenShareModal}
       >
-        <button
-          disabled={!hasApiKey || !validApiKey || !hasStore}
-          className={classNames(
-            "relative inline-flex h-full w-full items-center justify-center gap-[4px] bg-muted px-5 py-3 text-sm font-semibold text-foreground transition-all duration-150 ease-in-out hover:bg-background hover:bg-hover",
+        <ShadTooltip
+          content={
             !hasApiKey || !validApiKey || !hasStore
-              ? "button-disable text-muted-foreground"
-              : "",
-          )}
-          data-testid="shared-button-flow"
+              ? "Store API Key Required"
+              : ""
+          }
+          side="bottom"
+          align="end"
         >
-          <ForwardedIconComponent
-            name="Share3"
+          <button
+            disabled={!hasApiKey || !validApiKey || !hasStore}
             className={classNames(
-              "-m-0.5 -ml-1 h-6 w-6",
+              "relative inline-flex h-full w-full items-center justify-center gap-1.5 rounded px-3 py-1.5 text-sm font-semibold text-foreground transition-all duration-150 ease-in-out hover:bg-accent",
```

Comment:
can we move to a css class on the applies.css file just to improve the readability of this component?